### PR TITLE
Refactor the Awesome-book Website

### PR DESCRIPTION
**Pull request by DaveZag**
# Refactor the Awesome-book Website
### On this branch we:
- Refactor the awesome-book project to use modules.
- Put every function in its module.
- Use import and export to connect files and reuses them when needed.
- Use the Luxon library to get the time and display it in the top-right corner.
- Run and Fix all linters errors if any.
- Ignore all unnecessary files in .gitignore. 🙃

### CODE REVIEW

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Review Request :red_circle: :traffic_light: :rotating_light:

## Hey @DaveZag :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**

## 1. Your Linters
- [ ] Your linters are not passing all the tests. :thinking: Kindly fix those errors because they would break your application. :fearful:

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:
a
Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DaveZag** [comment]
Hi @HAMMAS-SALEEM , The linters don't pass because of luxon.js which has more than 500 linters errors.
I think I'm gonna disable the linters on that file since it does not need to be checked.🤔

File: app/script/modules/show_section.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      for (let counter = 0; counter < sections.length; counter += 1) {
+        if (e.target.getAttribute('data-show') === sections[counter].id && sections[counter].classList.contains('show') !== true) {
+          sections[counter].className += ' show';
+        } else {
+          sections[counter].classList.remove('show');
+        }
+      }
+    });
+  }
+}
```
Comment (TedLivist) [inline app/script/modules/show_section.js:23]: - [x] I think there may be an issue with your application navbar. I am unable to navigate through the pages. As can see from the URL bar I have tried going through the pages but the display does not change.

- [x] Also your date and time from luxon is not displayed at all

  List page | Add new page | Contact page
  :---:|:---:|:---:
  htm-list | htm-add | htm-contact

  I think these issues may have to do with this:

  lu

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @DaveZag,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/script/modules/show_section.js (hunk 1 above)
Comment (DaveZag) [reply app/script/modules/show_section.js:23]: Thanks for the feedback. I'm going to fix that.🙃

Comment (DaveZag) [reply app/script/modules/show_section.js:23]: Hi @TedLivist, the bugs are fixed. A confirmation from your side would be appreciated.🙃

**Review by Wes-Isaac — APPROVED** [review]
Hi @DaveZag , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👀.
- Great PR description 👍.
- No linter error ✅.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @DaveZag , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👀.
- Great PR description 👍.
- No linter error ✅.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**